src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class to maintain the deadline tasks entered by user.
+ */
+public class Deadline extends Task {
+    protected LocalDate deadline;

```

Comment:
It was a little confusing differentiating between `deadline` and `endTime`. Perhaps you can consider renaming `deadline` to `endDate` instead so that it is more similar to `endTime`?